homeassistant/components/climate/melissa.py

```diff
@@ -60,30 +68,20 @@ def is_on(self):
         if self._cur_settings is not None:
             return self._cur_settings[self._api.STATE] in (
                 self._api.STATE_ON, self._api.STATE_IDLE)
-        else:
-            _LOGGER.info("Can't determine state of %s", self.entity_id)
-            return STATE_UNKNOWN
+        return False
```

Comment:
Is this state checked now in the main component, or what happens when an invalid or unknown state gets propagated here?

Reply:
is_on should only return a bool, so if the state is not present it defaults to off (false)

Reply:
You can return `None` here to indicate unknown state, if that is preferred.